### PR TITLE
DB and Tables created

### PR DESCRIPTION
**Pull request by ishimwepati**
Hi !

Our team of @AjroudiRida  and @ishimwepati, below are the features we implemented :

1. We created a Table called : ```schema_diagram```
2. We created the following tables based on the diagram 👍 

- Patients Table 

patients

- Invoices Table 

invoices

- Medical_history Table 

medical_histories

- Invoices_items 

invoice_items

- Treatment Table 

treatments

-Treatments_history

treatments_history

Below is the Joined table for Medical_histories and Treatment as their relation is many to many 👍

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+CREATE TABLE treatments_history (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    medical_history_id INT,
+    treatment_id INT,
+    FOREIGN KEY (medical_history_id)
+    REFERENCES medical_histories(id),
+    FOREIGN KEY (treatment_id)
+    REFERENCES treatments(id),
+    PRIMARY KEY (id)
+);
```
Comment (mukhammadrizooff) [inline schema_based_on_diagram.sql:69]: - [ ] [REQUIRED] Good job so far! All tables should have a `PRIMARY KEY` set
all `FOREIGN KEYS` are set (by using refrences table_name(id) in column definition and all `FOREIGN KEYS` should have `indexes` created 🙏🏼 

```sql
CREATE INDEX ON medical_histories (patient_id);
CREATE INDEX ON invoices (medical_history_id);
CREATE INDEX ON invoice_items (invoice_id);
CREATE INDEX ON invoice_items (treatment_id);
CREATE INDEX ON medical_histories_has_treatments (medical_history_id);
CREATE INDEX ON medical_histories_has_treatments (treatment_id);
```

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @TEAM, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- TEAM ✔️ 
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,2 +1,147 @@
...
+I would like to thank Microverse for the project inspiration.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```
Comment (Johnadibe) [inline README.md:144]: > [OPTIONAL] As this was missed by the previous reviewer, kindly ensure you fix your MIT link as it is broken.👍

**Review by Johnadibe — APPROVED** [review]
# Status: Approved ✔️
### Hi @ishimwepati and @AjroudiRida👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

LeonardoInGatsby

## Highlights
- Implemented all changes requested by the previous reviewer✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**